docs/getting-started/advanced-setting.asciidoc

```diff
@@ -189,7 +189,7 @@ The `securitySolution:maxUnassociatedNotes` field determines the maximum number
 [[exclude-cold-frozen-data-rule-executions]]
 == Exclude cold and frozen data from rule executions 
```

Comment:
Since we have two different methods of excluding certain data tiers from rules, I think we should change the title to be `Exclude cold and frozen data from all rules` for the advanced setting, vs `Exclude cold and frozen data from a single rule` for the manual filter approach. This way users that follow the link from the manual page to here immediately understand why we have multiple approaches.

Reply:
Updated the title for the advanced setting in [203b0bb](https://github.com/elastic/security-docs/pull/5962/commits/203b0bb557f03206a480f4c0b70f6f854079d202).